test/kt/esimd_radix_sort.cpp

```diff
@@ -134,6 +134,26 @@ test_sycl_iterators(sycl::queue q, std::size_t size, KernelParam param)
     EXPECT_EQ_RANGES(ref, input, msg.c_str());
 }
 
+template <typename T, bool IsAscending, std::uint8_t RadixBits, typename KernelParam>
+void
+test_buffer(sycl::queue q, std::size_t size, KernelParam param)
```

Comment:
`test_host_buffer` ?

Reply:
There is only `sycl::buffer`, so `test_host_buffer` would incur confusion. Perhaps, it is better to rename the function to `test_sycl_buffer`.